common/primitives/src/messages.rs

```diff
@@ -130,6 +139,43 @@ impl<T> BlockPaginationResponse<T> {
 	}
 }
 
+#[cfg(feature = "runtime-benchmarks")]
+/// A trait for helping setup state for running benchmarks.
+pub trait BenchmarkHelper<AccountId> {
```

Comment:
this helper is a bit weird since it is a mix of functionalities from two different pallets

Reply:
I agree. I did not want to spend more time fixing the bug. We still need to remove the duplication from the schemas. So whoever does that can improve this.